analysis/SConstruct

```diff
@@ -48,7 +48,9 @@ Default('./build', './release')
 if mode == 'cache':
     CacheDir(cache_dir)
 
-debrief_env = {'MAXIT' : 10,
+# Debrief args is a dictionary storing the arguments that are useful for functions to run 
+# after all Scons targets have been built. Scons_debrief will pass these args into those functions. 
+debrief_args = {'MAXIT' : 10,
```

Comment:
May be add some comment for `maxit`?

Reply:
Perhaps we want to make it close to `log`? Since they are used in the same function.